### DISCUSSION TITLE
selectable dates month model appears below the model where the component is located

### DISCUSSION THREAD

**Discussion by Yedidya10** [rsuite/rsuite#3321] [Q&A]
DateRangePicker integration in MUI

My app uses MUI as main UI library but I thought to add DateRangePicker from rsuitejs (since it's not free there)

It seems to work fine except that the model for the selectable dates month appears below the model where the component itself is located.

Is there a way to access the original DateRangePicker settings to set its model independently?

Thanks.

![image](https://github.com/rsuite/rsuite/assets/86730512/6ffb248e-b445-474c-8b2e-145fe2d64be9)

**Comment by SevenOutman** [comment]
The popup calendar panel of DateRangePicker sits at `z-index: 7` as `.rs-picker-menu` class defines. You could override this style with CSS.

If you use Less in your project, you could do this by simply overridding `@zindex-picker-menu` variable.

**Reply by Yedidya10** [reply]
Could you please elaborate more?

In my Next JS application I can use CSS models or emotion

I tried several ways;
```
<DateRangePicker
        size="lg"
        className={styles.dateRangePicker}
       placeholder="Select Date Range"
      style={{
        width: 260,
       display: 'block',
       marginBottom: 10,
       zIndex: 99999,
     }}
 />

// ItemRequestForm.module.scss
.dateRangePicker {
   .rs-picker-menu {
     z-index: 9999;
   }
}
```

**Reply by SevenOutman** [reply]
The popup element is not mounted within the DateRangePicker element, but under `<body>`. So you should override `.rs-picker-menu` styles at the root level rather than under `.dateRangePicker`.

```css
.rs-picker-menu {
  z-index: 9999;
}
```
